wrappers/python/pyrs_device.cpp

```diff
@@ -39,8 +39,12 @@ void init_device(py::module &m) {
         .def("__repr__", [](const rs2::device &self) {
             std::stringstream ss;
             ss << "<" SNAME ".device: " << self.get_info(RS2_CAMERA_INFO_NAME)
-                << " (S/N: " << self.get_info(RS2_CAMERA_INFO_SERIAL_NUMBER)
-                << ")>";
+                << " (S/N: " << self.get_info(RS2_CAMERA_INFO_SERIAL_NUMBER);
+            if (self.supports(RS2_CAMERA_INFO_FIRMWARE_VERSION))
+                ss << "  FW: " << self.get_info(RS2_CAMERA_INFO_FIRMWARE_VERSION);
+            if (self.supports(RS2_CAMERA_INFO_CAMERA_LOCKED))
+                ss << "  " << (self.get_info(RS2_CAMERA_INFO_CAMERA_LOCKED) ? "LOCKED" : "UNLOCKED");
```

Comment:
It escaped me, but then I saw that the LibCI device shows as "LOCKED":
`get_info` returns a `const char *` -- not a float! It will be `YES` or `NO`... so we can change our output to `LOCKED:<YES/NO>`